test/stdlib/TestUserInfo.swift

```diff
@@ -20,15 +20,29 @@ import StdlibUnittest
 class TestUserInfoSuper : NSObject { }
 #endif
 
-struct SubStruct {
+struct SubStruct: Equatable {
     var i: Int
     var str: String
+
```

Comment:
This looks like it might be changing what we're testing, since the default for classes (and thus, presumably, id-as-Any boxes) is to compare by object identity.
In any case, the commit message (at least) should describe what practical effect this change has.  Was the test checking the wrong thing?